larray/tests/test_array.py

```diff
@@ -706,108 +706,117 @@ def test_getitem_structured_key_with_groups():
 
 
 def test_getitem_single_larray_key_guess():
-    a = Axis(['a1', 'a2'], 'a')
-    b = Axis(['b1', 'b2', 'b3'], 'b')
-    c = Axis(['c1', 'c2', 'c3', 'c4'], 'c')
+    # TODO: we really need another way to get test axes, e.g. testaxes(2, 3, 4) or testaxes((2, 3, 4))
+    a, b, c = ndtest((2, 3, 4)).axes
+    arr = ndtest((a, b))
+    # >>> arr
+    # a\b b0 b1 b2
+    #  a0  0  1  2
+    #  a1  3  4  5
 
     # 1) key with extra axis
-    arr = ndtest([a, b])
-    # replace the values_axis by the extra axis
-    key = LArray(['a1', 'a2', 'a2', 'a1'], [c])
-    assert arr[key].axes == [c, b]
-
-    # 2) key with the values axis (the one being replaced)
-    arr = ndtest([a, b])
-    key = LArray(['b2', 'b1', 'b3'], [b])
+    key = LArray(['a0', 'a1', 'a1', 'a0'], c)
+    # replace the target axis by the extra axis
+    expected = from_string(r"""
```

Comment:
I'm wondering if this behavior is explained in the tutorial or not? If not, it must be explained.

Reply:
> I think updating the tutorial is out of scope for this PR though.
Sorry, my intention was not to ask you to update the tutorial in this PR but rather to suggest to remember that we need to update the tutorial in the future

Reply:
No problem, I didn't think that either :)